python/lsst/ctrl/bps/defaults.py

```diff
@@ -0,0 +1,37 @@
+# This file is part of ctrl_bps.
```

Comment:
From the hindsight the better name for this module would be `constants.py`. The intention behind creating it was to have a place for symbolic constants (which are not supposed to be modified by the user) to avoid "magic numbers" in the code. I will rename it to `constants.py` and add a docstring explaining what this module is for.